extra_data/reader.py

```diff
@@ -1871,8 +1889,25 @@ def open_run(
         run = index(run)  # Allow integers, including numpy integers
     run = 'r' + str(run).zfill(4)
 
-    return RunDirectory(
+    dc = RunDirectory(
         osp.join(prop_dir, data, run), include=include, file_filter=file_filter,
         inc_suspect_trains=inc_suspect_trains, parallelize=parallelize,
         _use_voview=_use_voview,
     )
+
+    # Normalize string arguments to be an absolute Path
+    if isinstance(aliases, str):
+        if len(list(string.Formatter().parse(aliases))) > 1:
+            aliases = aliases.format(prop_dir)
```

Comment:
Why check for the presence of formatting placeholders and not just run it through `str.format`? :thinking:

Reply:
Huh, TIL that format doesn't care if there aren't any format arguments :eyes: Fixed in 4794482.